tests/s3_bucket/test_github_content_modules_1.py

```diff
@@ -55,9 +59,45 @@ def test_github_content_modules_1(git_content_repos_1, headers_data):
 
                     resp_content = module_resp.text
 
-                    # Verifies index.cnxml files for presence of content
-                    assert (
-                        resp_content.count("<md:") > 1
-                        and resp_content.count("<content") >= 1
-                        and resp_content.count("<title") >= 1
-                    )
+                    soup = BeautifulSoup(resp_content, "xml")
+                    document = soup.find_all("document")
+                    content = soup.find_all("content")
+
+                    for doc in document:
+
+                        # Verifies index.cnxml for presence of md:title and md:uuid metadata
+                        try:
+                            md_title = re.findall("<md:title>(.*)</md:title>", str(doc))
```

Comment:
Why use regex when you already have `BeautifulSoup`? This could be `assert doc.find("md:title")`. Also, the standard `re.findall` will return an empty array when it does not find anything, not `None`. So it will never be `None`.

Reply:
you are right... on it